### PR TITLE
feat: definitions.d directory

### PR DESCRIPTION
**Pull request by boredland**
- move readmes into readme.md files
- make defintions a directory so we can easier update the skel after updated wthout loosing userspace customizations

### CODE REVIEW

File: community/sway/etc/sway/config.d/99-autostart-applications.conf [hunk 1]
```diff
@@ -21,3 +21,8 @@ exec_always {
     '[ -x "$(command -v sworkstyle)" ] && pkill sworkstyle; sworkstyle &> /tmp/sworkstyle.log'
     '[ -x "$(command -v playerctl)" ] && pkill playerctl; playerctl -a metadata --format \'{{status}} {{title}}\' --follow | while read line; do pkill -RTMIN+5 waybar; done'
 }
+
+# https://github.com/Alexays/Waybar/issues/1093#issuecomment-841846291
```
Comment (FallenChromium) [inline community/sway/etc/sway/config.d/99-autostart-applications.conf:25]: Wow, did we even have this issue? I still don't fully get what's wrong after reading through the issue

**Comment by FallenChromium** [comment]
Nice PR! Overall I support the idea. It's nice that we don't rewrite the definitions and userfiles by default. What about moving readme.md to a more visible place, such as ~/.config/sway/readme.md?

File: community/sway/etc/sway/config.d/99-autostart-applications.conf (hunk 1 above)
Comment (boredland) [reply community/sway/etc/sway/config.d/99-autostart-applications.conf:25]: I had on one machine, not the other. I think its a specific combination of pipewire and machine. Don't really know. But as I see it, it is harmless for everybody else, so I though we can share it with everybody. the video in that issue shows the behaviour exactly how it is on my work-machine.

**Comment by boredland** [comment]
> Nice PR! Overall I support the idea. It's nice that we don't rewrite the definitions and userfiles by default. What about moving readme.md to a more visible place, such as ~/.config/sway/readme.md?

we could, but I actually liked the idea of putting those readme to "where they belong" so we don't have to explain that part.

**Pull request merged**